### PR TITLE
remove Contribook stuff and add central 

### PR DESCRIPTION
**Pull request by Kawohl**
remove Contribook stuff and add central

### CODE REVIEW

**Comment by Kawohl** [comment]
@jnweiger @lefherz

File: page-contact.php [hunk 1]
```diff
@@ -1,38 +1,11 @@
...
-		echo($data['role']);
-
-		echo('</td></tr>');
-	}
-
-	echo('</table>');
-
-}
-?>
 Note that, unfortunately, most of our contributors have not (yet) set up a blog and and added themselves here. You can find a much more complete list of past and present ownCloud contributors on <a href="https://github.com/orgs/owncloud/people">github</a>, which is where we develop ownCloud.
```
Comment (lefherz) [inline page-contact.php]: The last 2 paragraphs have to go, too; you are deleting their context^^

File: page-contact.php [hunk 2]
```diff
@@ -1,38 +1,11 @@
...
 
 If you want to be added to the blog roll, please ping us via <a href="mailto:ownclouders@owncloud.com">E-Mail</a>,
```
Comment (lefherz) [inline page-contact.php]: So this has to be deleted, too :)

Comment (Kawohl) [reply page-contact.php]: No we want to have something else there in the new owncloud.org. To have a owncloud related blogroll. For that we need to be in contact.

**Comment by lefherz** [comment]
@agssl was this double-checked on staging? o.0

**Pull request merged**